main.go

```diff
@@ -41,6 +46,8 @@ func main() {
 	)
 	flag.BoolVar(&cl2Flag, "cl2", false, "store output in CL2 format")
 	flag.BoolVar(&cl2ArchiveFlag, "cl2_archive", false, "store output in CL2 archive format")
+	flag.BoolVar(&useCIE2000, "c", false, "use CIE Delta E 2000 instead of Euclidean")
```

Comment:
I agree, I think that's a good way to keep consistent with things. It would make it much easier if other algorithms are added in the future like CIE DE 1976 and such.